src/seedu/addressbook/AddressBook.java

```diff
@@ -210,6 +222,13 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
+        getUserCommand();
+    }
+
+    /**
+     * Continuously gets user input 
+     */
+    private static void getUserCommand() {
```

Comment:
Good job applying `SLAP`.
The comment can be improve further.
`Gets user inputs and displays back the results` would be more appropriate to the implementation of the method.